rust-src/crypto_common/src/impls.rs

```diff
@@ -194,7 +194,7 @@ impl Deserial for Signature {
     fn deserial<R: ReadBytesExt>(source: &mut R) -> ParseResult<Self> {
         let mut buf = [0u8; SIGNATURE_LENGTH];
         source.read_exact(&mut buf)?;
-        Ok(Signature::new(buf))
+        Ok(Signature::from(buf))
```

Comment:
What about using either `from_bytes` or `try_from`. `from` will panic if the signature bytes are invalid.

Reply:
No, it won't panic. It is the identity function since Signature is [u8; 64].

Reply:
So in 1.0 it is the identity function, but they changed the semantics in 1.3 apparently. Awesome.

Reply:
I'm using from_bytes now.

I believe this does not change the semantics anywhere else in the project.